Initial BIPOC slack draft (#1273)
writethedocs/www: docs/bipoc.rst

Our #bipoc channel
==================

> This is a space dedicated to supporting BIPOC members within our community.

Reviewer: Perhaps we could explain what "BIPOC" means here? I know, but not everyone who runs into this page will know.

Reviewer 2: I agree! One of my friends raised the same point. We'll add an explanation here or somewhere close to the top of the page.

Author: ```suggestion
This is a space dedicated to supporting BIPOC (Black, Indigenous, and People of Color) members within our community.
```

> This is a space dedicated to supporting BIPOC (black, Indigenous and people of color) members within our community.

Author: ```suggestion
This is a space dedicated to supporting BIPOC (black, indigenous and people of color) members within our community.
```

Our #bipoc channel
==================

This is a space dedicated to supporting BIPOC tech writers/software development/people in open source. 

Background
----------

As with many other professions, BIPOC are painfully underrepresented in both technical communication and software development.

> <!-- SOME STATS WOULD BE USEFUL HERE. -->

Author: ```suggestion
Here's how you'd propose substitute text
```

Background
----------

As with many other professions, BIPOC remain painfully underrepresented in the tech industry, including but not limited to technical communication, software development, and open source.

According to Open MIC's 2017 Report, `Breaking the Mold: Investing in Racial Diversity in Tech`_:

.. _`Breaking the Mold: Investing in Racial Diversity in Tech`: http://breakingthemold.openmic.org/

* Black people, Latinos and Native Americans are underrepresented in tech by 16-to-18 percentage points compared with their presence in the U.S. labor force overall. Black people and Latinos each comprise just 5.3 percent of the Professionals category in U.S. tech industry labor data.
* While Asians are represented at a higher rate in the tech workforce than the private sector overall, white people are 1½ times more likely than Asians to rise to an executive rank.
* Among people of color who do enter the industry, many report isolation, discrimination and toxic work environments that prompt them to take their talent else-where. People of color leave tech at more than 3.5 times the rate of white men.

> To shrink racial disparities and to build equity, we offer this space as BIPOC :doc:`/documentarians`. (which includes technical writers) to:

Reviewer 2: ```suggestion
To shrink racial disparities and to build equity, we offer this space as BIPOC :doc:`/documentarians` (which includes technical writers) to:
```

I think we could remove the entire parenthetical comment. But for now, let's at least remove that period. 👍

Author: Yup, I included the parenthetical because of confusion from some on the difference between TWs and documentarians

Background
----------

As with many other professions, BIPOC remain painfully underrepresented in the tech industry, including but not limited to technical communication, software development, and open source.

According to Open MIC's 2017 Report, `Breaking the Mold: Investing in Racial Diversity in Tech`_:

.. _`Breaking the Mold: Investing in Racial Diversity in Tech`: http://breakingthemold.openmic.org/

* Black people, Latinos and Native Americans are underrepresented in tech by 16-to-18 percentage points compared with their presence in the U.S. labor force overall. Black people and Latinos each comprise just 5.3 percent of the Professionals category in U.S. tech industry labor data.
* While Asians are represented at a higher rate in the tech workforce than the private sector overall, white people are 1½ times more likely than Asians to rise to an executive rank.
* Among people of color who do enter the industry, many report isolation, discrimination and toxic work environments that prompt them to take their talent else-where. People of color leave tech at more than 3.5 times the rate of white men.

> To shrink racial disparities and to build equity, we offer this space as BIPOC tech writers to:

Reviewer 3: Made a comment in the Slack channel as well, but wanted to ask about `tech writers` vs., say, `documentarians`. The latter would be more consistent with how we refer to ourselves across WtD. _But_, to Cheryl's point from yesterday, those who aren't already in the WtD community may not be familiar with the term and want to better understand who we are and what we all do. 

Hm, I suppose we could also link out to the [Documentarians](https://www.writethedocs.org/documentarians/) page for their reference?

Author: ```suggestion
To shrink racial disparities and to build equity, we offer this space as BIPOC :doc:`/documentarians`. (which includes technical writers) to:
```

In a global field that is largely represented by `white`_ people, admonishing racism in spaces full of non-BIPOC people is limited in its impact. For one thing, these spaces do not capture the complexity of experiences that BIPOC have lived. BIPOC individuals and communities often experience various `intersections`_ of both privilege (e.g., `colorism`_) and discrimination (e.g., ability, gender identity, sexuality, citizenship status, neuro-status, linguistic `fluency`_).
In a global field that is largely represented by `white`_ people, admonishing racism in spaces full of non-BIPOC people is limited in its impact. For one thing, these spaces do not capture the complexity of experiences that BIPOC have lived. BIPOC individuals and communities often experience various `intersections`_ of both privilege (e.g., `colorism`_) and discrimination (e.g., physical/cognitive ability, gender identity, sexuality, citizenship status, neuro-status, socioeconomic status, linguistic `fluency`_, religion, ethnicity).

> .. _white: https://nmaahc.si.edu/sites/default/files/styles/image_caption/public/images/captioned/whiteculture_info_1.png

Reviewer 4: The internet's a terrible place, so the infographic that used to be at this link got co-opted by racists being like "this sounds great!" - so I changed the link to go to the page where that infographic used to be.

Reviewer 2: Thank you so much!!!! Another friend pointed this out to me and I was wondering what had happened to it ...

As a side note: I actually saved that infographic to my computer a while back. We could probably use it on the page, but I think it's better and less obtrusive to use the link ref that you updated. 👍

Reviewer 4: No problem, I'm glad I was able to catch it!

I was able to find the infographic on the Wayback Machine, so we could still make a hyperlink pretty easily through there if we wanted to. But I think the co-opting problem points to a greater issue with the infographic: if people are so indoctrinated to accept cultural white supremacy, they won't see anything wrong with what it says. So maybe it won't hurt to have that more basic information available in case people need it.
